packages/hadron-document/src/element.ts

```diff
@@ -699,7 +699,7 @@ export class Element extends EventEmitter {
    */
   revert(): void {
     if (this.isAdded()) {
-      this.parent!.elements!.remove(this);
+      this.parent?.elements?.remove(this);
```

Comment:
just a question, in which case do we end up with a document that doesn't exist?

Reply:
i tried really hard to find the root cause, but was not. I am giving up and will merge this PR.